server/controllers/mobile-phone-token-controller.js

```diff
@@ -61,6 +61,57 @@ export const createMobilePhoneToken = (req, res) => {
   }
 };
 
+export const mobilePhoneTokenStatus = (req, res) => {
+  const reqOrg = req.params.organization;
+  const validSlug = config.some((org) => {
+    if (org.slug === reqOrg) {
+      // merge default config and custom config
+      const conf = merge(defaultConfig, org);
+      const {host} = conf;
+      const url = reverse("mobile_phone_token_status", getSlug(conf));
+      const timeout = conf.timeout * 1000;
+      // make AJAX request
+      axios({
+        method: "get",
+        headers: {
+          "content-type": "application/x-www-form-urlencoded",
+          Authorization: req.headers.authorization,
+          "accept-language": req.headers["accept-language"],
+        },
+        url: `${host}${url}/`,
+        timeout,
+      })
+        .then((response) => {
+          res
+            .status(response.status)
+            .type("application/json")
+            .send(response.data);
+        })
+        .catch((error) => {
+          logResponseError(error);
+          // forward error
+          try {
+            res
+              .status(error.response.status)
+              .type("application/json")
+              .send(error.response.data);
+          } catch (err) {
+            res.status(500).type("application/json").send({
+              response_code: "INTERNAL_SERVER_ERROR",
+            });
+          }
+        });
+    }
+    return org.slug === reqOrg;
+  });
+  // return 404 for invalid organization slug or org not listed in config
+  if (!validSlug) {
+    res.status(404).type("application/json").send({
+      response_code: "INTERNAL_SERVER_ERROR",
```

Comment:
Probably we don't want to send "INTERNAL_SERVER_ERROR" here but something different.
If this case is triggered, we must make sure the old behavior of sending the SMS token anyway is not triggered in this case because we know it's not needed.